package.json

```diff
@@ -23,6 +23,7 @@
   },
   "homepage": "https://github.com/EddieJaoudeCommunity/EddieBot#readme",
   "dependencies": {
+    "@types/soundex-code": "^1.0.0",
```

Comment:
I think these dependencies ("@types/....") usually go to devDependencies. Since they are not needed for production, for example.
Could we change this @RaisinTen ?

Reply:
@BOLT04 sure! :smiley: